linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -47,18 +51,23 @@
 
   public static void main(String[] arguments)
       throws RepositoryException, IOException, LinkageMonitorException {
-    if (arguments.length < 1) {
+    if (arguments.length < 1 || arguments[0].split(":").length != 2) {
       System.err.println(
-          "Please specify BOM coordinates. Example: com.google.cloud:libraries-bom:1.2.1");
+          "Please specify BOM coordinates without version. Example:"
+              + " com.google.cloud:libraries-bom");
       System.exit(1);
     }
     String bomCoordinates = arguments[0];
-    new LinkageMonitor().run(bomCoordinates);
+    List<String> coordinatesElements = Splitter.on(':').splitToList(bomCoordinates);
+
+    new LinkageMonitor().run(coordinatesElements.get(0), coordinatesElements.get(1));
   }
 
-  private void run(String bomCoordinates)
+  private void run(String groupId, String artifactId)
       throws RepositoryException, IOException, LinkageMonitorException {
-    Bom baseline = RepositoryUtility.readBom(bomCoordinates);
+    String latestBomCoordinates = findLatestCoordinates(groupId, artifactId);
+    System.out.println("BOM Coordinates: " + latestBomCoordinates);
+    Bom baseline = RepositoryUtility.readBom(latestBomCoordinates);
```

Comment:
Possibly I should have kept the method that reads from an Artifact. We're doing a lot of back and forth between artifacts and BOM coordinates.